tests/test_deltafetch.py

```diff
@@ -86,8 +87,8 @@ def test_spider_opened_existing(self):
         mw.spider_opened(self.spider)
         assert hasattr(mw, 'db')
         assert isinstance(mw.db, type(dbmodule.db.DB()))
-        assert mw.db.items() == [('test_key_1', 'test_v_1'),
-                                 ('test_key_2', 'test_v_2')]
+        assert mw.db.items() == [(b'test_key_1', b'test_v_1'),
+                                 (b'test_key_2', b'test_v_2')]
```

Comment:
This is strange: `test_v_1` values are returned as bytes. But were inserted as strings (`db.put(b'test_key_1', 'test_v_1')`). Not sure if this is bsddb3 not casting back to int/string on fetch or what.